src/main/java/com/sparta/popupstore/domain/reservation/controller/ReservationController.java

```diff
@@ -1,12 +1,27 @@
 package com.sparta.popupstore.domain.reservation.controller;
 
+import com.sparta.popupstore.domain.common.annotation.AuthUser;
+import com.sparta.popupstore.domain.reservation.entity.Reservation;
 import com.sparta.popupstore.domain.reservation.service.ReservationService;
+import com.sparta.popupstore.domain.user.entity.User;
 import lombok.RequiredArgsConstructor;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDateTime;
 
 @RestController
 @RequiredArgsConstructor
+@RequestMapping("/popupstores")
 public class ReservationController {
 
     private final ReservationService reservationService;
+
+    @PostMapping("/{popupStoreId}/reservation")
+    public ResponseEntity<Reservation> reservePopupStore(
+            @PathVariable Long popupStoreId,
+            @AuthUser User user,
+            @RequestParam LocalDateTime reservationAt) {
```

Comment:
```suggestion
            @RequestBody reservationCreateRequestDto requestDto) {
```
조회가 아니면 RequsetParam은 쓰지 않는 게 좋을 것 같습니다.

Reply:
수정했습니다 :)